### ISSUE TITLE
Clearer Template Section, JSX support mention

### ISSUE THREAD

**Issue by sdras** [vuejs/vuejs.org#2082]
We need to explain a bit better that the most common way to write Vue components are SFC, and not template strings. We also need to mention earlier that we support JSX. I'm just hearing so much feedback based on misunderstanding here that this needs to be more clear. Happy to make a PR for it. 

@chrisvfritz @phanan @NataliaTepluhina for input.

**Comment by NataliaTepluhina** [comment]
@sdras I completely agree we need to focus on SFC more and mention template strings as a valid but not primary option.

Also agree on JSX, it really helps people with previous React experience to start with Vue easier.

Please let me know if you need any help with PR! 🙌

**Comment by mayeyun** [comment]
> 我们需要更好地解释一下，编写Vue组件的最常用方法是SFC，而不是模板字符串。我们之前还需要提一下，我们支持JSX。我只是听到这么多基于误解的反馈，这需要更清楚。很高兴为它做一个公关。
> 
> @chrisvfritz @phanan @NataliaTepluhina进行输入。
我是小白我也不太会刚入行不就把！
